install/docker/index.markdown

```diff
@@ -59,5 +59,5 @@ To make the source container more useful for developing and testing code, we rec
 
 MoveIt!'s docker containers are built automatically on dockerhub.com, but you can modify and build locally if desired with the following command:
 
-    cd moveit/.docker/kinetic/source
-    docker build -t moveit/moveit:kinetic-source .
+    cd moveit/.docker/source
+    docker build -t moveit/moveit:melodic-source .
```

Comment:
Do you want to leave this tagged as "... :melodic-source" and not just "... :source". It seems like it could be okay as is because the base image in source is melodic, but this is not explicit given the dir change above.

Probably not worth too much thought :)